libraries/AP_OSD/AP_OSD_ParamSetting.cpp

```diff
@@ -139,7 +139,7 @@ static const char* SERIAL_PROTOCOL_VALUES[] = {
     "FSKY_TX", "LID360", "", "BEACN", "VOLZ", "SBUS", "ESC_TLM", "DEV_TLM", "OPTFLW", "RBTSRV",
     "NMEA", "WNDVNE", "SLCAN", "RCIN", "MGSQRT", "LTM", "RUNCAM", "HOT_TLM", "SCRIPT", "CRSF",
     "GEN", "WNCH", "MSP", "DJI", "AIRSPD", "ADSB", "AHRS", "AUDIO", "FETTEC", "TORQ",
-    "AIS", "CD_ESC", "MSP_DP", "MAV_HL", "TRAMP", "DDS", "IMUOUT", "IQ", "PPP",
+    "AIS", "CD_ESC", "MSP_DP", "MAV_HL", "TRAMP", "DDS", "IMUOUT", "IQ", "PPP", "IBUS_TLM"
```

Comment:
We try and make these 6 chars so that they fit in the various display contexts

Reply:
Would `IBUSTLM` work? There are a few 7char ones. I feel like the "TLM" is important because not all serial protocols are telemetry.

Reply:
If there are 7 char ones already then ignore me

Reply:
7 is the longest so far. `IBUS_TLM` is 8 which would be longer. I can make it `IBUSTLM` but that breaks convention with the other telemetry ones which all use an underscore. Happy to do whichever you prefer!